client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -166,7 +167,9 @@ export default function SitesTableRow( { site }: SiteTableRowProps ) {
 						/>
 					</Column>
 					<Column mobileHidden>{ site.plan.product_name_short }</Column>
-					<Column mobileHidden>July 16, 1969</Column>
+					<Column mobileHidden>
+						<TimeSince date={ site.options?.updated_at } />
```

Comment:
Even though `<TimeSince>` is implemented in plain JavaScript, TypeScript seems to be inferring the props that are required and showing you the error. That's pretty fancy! Except it's inferring the types incorrectly 😄

This can be fixed by heading to `client/components/time-since/index.jsx` and documenting the types using the JSDoc method. Paste the following directly above the `TimeSince` definition.

```js
/**
 * @typedef {Object} TimeSinceProps
 * @property {string} date the timestamp to render
 * @property {string=} dateFormat Moment format string for the tooltip, default = 'll'
 * @property {string=} className forwarded to underlying <time> element
 */

/**
 * @param {TimeSinceProps} props
 */
```

This makes the `date` prop required. That means there's still a type error in `<SitesTableRow>` but I think that's correct. How should we handle the situation where we don't have the `updated_at` field? Maybe render a dash instead of the time since?

Reply:
Nice TIL! I've just added that parameter definition to the TimeSince component, and it works fine now.

I've adjusted the type check condition too, so now we pass a string to TimeSince and use a dash otherwise.